### DISCUSSION TITLE
where to configure mail ? and where to set melpa archives ?

### DISCUSSION THREAD

**Discussion by ghost** [Lambda-Emacs/lambda-emacs#11] [General]
setting lem-setup-mail.el I got this

Error (use-package): mu4e/:init: Could not find package mu4e-thread-folding. Updating recipe repositories: (org-elpa melpa gnu-elpa-mirror nongnu-elpa el-get emacsmirror-mirror) with ‘straight-pull-recipe-repositories’ may fix this Disable showing Disable logging

**Comment by ghost** [comment]
this is the line that gives the error 
https://github.com/wereket/lambda-emacs/blob/main/.local/lambda-library/lambda-setup/lem-setup-mail.el#L57

**Reply by mclearc** [reply]
Be sure you're following these directions: https://github.com/jwiegley/use-package#extending-the-load-path. Easiest is to see how the use-package macro expands so that you can get a sense of what garbage it is spitting out.

**Comment by ghost** [comment]
Also creating the new mail module in .local/lambda-setup folder did not ask me for auto insertion of skeleton module creation

**Reply by mclearc** [reply]
Thanks. Yeah I've been slow to move the mail setup over to the general config because there are so many ways it can go wrong. Will take a look at all this later tonight

**Comment by ghost** [comment]
I am sorry to pressure you, it is my impatient foolishness.
mail and org-mode config org being the piece de resistance or amuse-bouche or both, 
when all the modules will be ready, I will read mastering emacs book, and start learning

**Reply by mclearc** [reply]
No problem! Am very glad to hear you're finding lambda helpful :)

**Reply by mclearc** [reply]
Have you looked at my mail and org configs? 

mail: https://github.com/mclear-tools/dotemacs/blob/master/cpm-setup-email.el
org: https://github.com/mclear-tools/dotemacs/blob/master/cpm-setup-org.el

**Comment by mclearc** [comment]
> setting lem-setup-mail.el I got this
> 
> Error (use-package): mu4e/:init: Could not find package mu4e-thread-folding. Updating recipe repositories: (org-elpa melpa gnu-elpa-mirror nongnu-elpa el-get emacsmirror-mirror) with ‘straight-pull-recipe-repositories’ may fix this Disable showing Disable logging

I would use https://github.com/rougier/mu4e-folding over mu4e-thread-folding (same developer, newer package)

**Comment by mclearc** [comment]
> Updating recipe repositories: (org-elpa melpa gnu-elpa-mirror nongnu-elpa el-get emacsmirror-mirror) with ‘straight-pull-recipe-repositories’ may fix this Disable showing Disable logging

I just work with the default straight settings and that seems to work fine.
